OOP/Assignment1/PersonTest.java

```diff
@@ -0,0 +1,109 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PersonTest {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void testGetUid1() {
+		Person person = new Person(101, "udit");
+		assertEquals(101, person.getUid());
+	}
+
+	public void testGetUid2() {
```

Comment:
You don't require too many tests to test "getUid()" as setting of Uid did not involve much calculations, and we know that if this method works fine for UID=100 then it will work fine for any number.